pydicer/convert/pet_to_nifti_convert_new.py

```diff
@@ -0,0 +1,409 @@
+import warnings
+from os.path import join
+from datetime import time, datetime
+
+import numpy as np
+from skimage.draw import polygon
+import SimpleITK as sitk
+import pydicom as pdcm
+from pydicom.tag import Tag
+
+
+def convert_dicom_to_nifty(input_filepaths,

```

Comment:
This file should be named something short and easy to import. I recommend `pet.py`

nifti is spelt with an i, not y.